modular_codegen/README.md

```diff
@@ -14,11 +14,21 @@ dev_dependencies:
   build_runner: any
 ```
 
+If you use want to nullsafety with flutter2, use a nullsafety varsion.
```

Comment:
Update that line with: "If you want to use null-safety with flutter2, use a null-safety version"